backend/server/events/events.go

```diff
@@ -19,68 +19,99 @@ import (
 
 // FB response expects data (array of events), and paging, which we ignore.
 // However, should FB provide an error, we capture it.
+
+// FbResponse - Facebook API response container
 type FbResponse struct {
```

Comment:
Can we put this all into one type definition
```
type (
  FbResponse struct {
      Data  []FbRespEvent `json:"data"`
      Cover FbRespCover   `json:"cover"`
      Error FbRespError   `json:"error"`
  } etc.
)
``` 
Up to you whether this is something you want to do. Personally i just think it looks nicer.

Reply:
You're right, it does. Done!